moler/device/abstract_device.py

```diff
@@ -167,3 +191,22 @@ def start(self, cmd_name, **kwargs):
         :param kwargs: dict with parameters for command constructor.
         :return: command object
         """
+
+    def register_handler_to_notify_to_forget_device(self, handler):
+        """
+        Registers handler to notify when device should be forgot.
+
+        :param handler: callable to call when device should be forgot.
+        :return: None
+        """
+        if handler not in self._forget_handlers:
+            self._forget_handlers.append(handler)
+
+    def close_and_forget(self):
+        """
+        Closes device, if any command or device is attached to this device they will be finished.
+
```

Comment:
"or device is attached" - shouldn't it be "or event is attached"

Reply:
Yes :)